### ISSUE TITLE
Bug: ATT&CK heatmap dashboard widget not showing

### ISSUE THREAD

**Issue by vincenzocaputo** [MISP/MISP#9189]
### Expected behavior

The ATT&CK heatmap dashboard widget is displayed.

### Actual behavior

The widget remains blank

![immagine](https://github.com/MISP/MISP/assets/32276363/c243c947-e410-4880-8c93-5c5d4c626655)

### Steps to reproduce

Open Dashboards. Add ATT&CK Heatmap widget.

It seems the bug has been introduced in the latest release.

### Version

2.4.173

### Operating System

RedHat

### Operating System version

Red Hat Enterprise Linux

### PHP version

7.4.30

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ahmedxuae91** [comment]
Yes, I am facing same issue as well. Seems it is a bug in 2.4.173

**Comment by davidcbbc** [comment]
Same issue here , any fix for this?

**Comment by davidcbbc** [comment]
Just found out the problem ,  you have to write the filters so the heatmap can be rendered , like this:

```json
{
    "filters": {
        "timestamp": [
            "2023-07-15",
            "2023-09-22"
        ],
        "published": [
            "0",
            "1"
        ]
    }
}
```

Don't forget to refresh the browser (it takes a while to render)

**Comment by vincenzocaputo** [comment]
Thank you @davidcbbc. With this workaround it works fine. Actually, the timestamp filter seems to be enough.
